.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    "env",
+    "react"
+  ],
+  "plugins": [
+    "transform-class-properties"
+  ]
+}
```

Comment:
I think it makes sense to use the same `.babelrc` configuration from the foreman.
At production build time, it will use them anyway.